pyproject.toml

```diff
@@ -28,10 +28,11 @@ readme = 'README.md'
 version = '0.1.0'
 
 [tool.poetry.dependencies]
-pyfiglet = '>=0.8.0'
```

Comment:
I have removed pyfiglet as only and old version is compatible with Python 3.8, and there is no added value adding this dependency